harmony-one/x
# User Profile and Data Feed from "Talk to Me!"

TODO:
- [ ] Test coverage for the whole code
- [ ] Ensure data is fetched from all sources not a single source:
  - Update `getData` to not accept `urlString`
  - Create a `sources` variable that contains all the sources string url
  - Update `getData` to fetch and append data from `sources`
